src/components/attribute-filter.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { useStateContext } from "../hooks/use-state";
+import { AttrType, dailyMonthlyAttrMap, hourlyAttrMap } from "../types";
+import EditIcon from "../assets/images/icon-edit.svg";
+
+import "./attribute-filter.scss";
+
+export const AttributeFilter = () => {
+  const {state} = useStateContext();
+  const frequency = state.frequency;
+  const units = state.units;
+  const attributes = state.attributes;
```

Comment:
Is there a reason to re-assign these instead of using destructuring? I.e. `const {frequency, units, attributes} = state`

Reply:
Hah! Brain fog.